src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
```

Comment:
I like how the the ordering of import statements in all of your classes are consistent, making it easier to browse the list and determine the dependencies